command/vendored_plugins.go

```diff
@@ -87,26 +69,14 @@ var VendoredBuilders = map[string]packersdk.Builder{
 	"azure-arm":           new(azurearmbuilder.Builder),
 	"azure-chroot":        new(azurechrootbuilder.Builder),
 	"azure-dtl":           new(azuredtlbuilder.Builder),
-	"cloudstack":          new(cloudstackbuilder.Builder),
 	"docker":              new(dockerbuilder.Builder),
 	"googlecompute":       new(googlecomputebuilder.Builder),
-	"hcloud":              new(hcloudbuilder.Builder),
-	"hyperv-iso":          new(hypervisobuilder.Builder),
-	"hyperv-vmcx":         new(hypervvmcxbuilder.Builder),
-	"hyperone":            new(hyperonebuilder.Builder),
-	"jdcloud":             new(jdcloudbuilder.Builder),
-	"lxc":                 new(lxcbuilder.Builder),
-	"lxd":                 new(lxdbuilder.Builder),
-	"ncloud":              new(ncloudbuilder.Builder),
-	"openstack":           new(openstackbuilder.Builder),
 	"proxmox":             new(proxmoxiso.Builder),
```

Comment:
Is there a reason why proxmox remains here? Since it's a community plugin, I figured this would be removed as well

Reply:
Glad you called it out. It is pretty heavily used so I was hesitant.  Seeing how out of date it is I can remove.  Any thoughts before I pull the trigger on removing it?

Reply:
I think we can remove it here too, if we call out that it won't be bundled in anymore I think it's good!